src/ds/ds5/ds5-color.cpp

```diff
@@ -9,6 +9,7 @@
 #include "proc/color-formats-converter.h"
 #include "ds5-color.h"
 #include <rsutils/string/from.h>
+#include "common/os.h"
```

Comment:
Let's add this function here instead:
https://github.com/IntelRealSense/librealsense/tree/development/third-party/rsutils/include/rsutils/os.h

Noa is already doing it on AUS so it will merge together later

Reply:
OK